avogadro/core/color3f.h

```diff
@@ -81,21 +82,15 @@ class Color3f
   const float* data() const;
 
 protected:
-  float m_data[3];
+  std::array<float, 3> m_data;
 };
 
-inline Color3f::Color3f(float r, float g, float b)
-{
-  m_data[0] = r;
-  m_data[1] = g;
-  m_data[2] = b;
-}
+inline Color3f::Color3f(float r, float g, float b) : m_data({ r, g, b }) {}
 
 inline Color3f::Color3f(int r, int g, int b)
+  : m_data({ static_cast<float>(r) / 255.0f, static_cast<float>(g) / 255.0f,
```

Comment:
Did `clang` format this? It's one place I might lean to 3 separate lines for readability. 

(I'm also wondering if 255.0f should be a declared constant somewhere also for readability.)

Reply:
Yea I ran clang formatting over the commit with git and made all the changes posted by the formatting bot. 

I can absolutely extract it out into a constant and store it in the source file.